config/config.yaml

```diff
@@ -188,14 +188,6 @@ branch-protection:
           branches:
             master:
               protect: true
-          required_status_checks:
```

Comment:
Since we're here... :)

Should we insert some ProwJob as a `required_check` to avoid PRs getting merged without checks other than approval?

Reply:
I think at least `check-prow-config` would be a good idea. I can go ammend that

Reply:
Added 3 here

```
          required_status_checks:
            contexts:
              - "ci/circleci: deploy/terraform"
              - "ci/circleci: deploy/prow"
              - "check-prow-config"
```

WDYT?

Reply:
💯